app/views/items/index.html.slim

```diff
@@ -1,7 +1,12 @@
-- @items.each do |item|
-  /.swiper-slide
-  .item
-    img.image [src="#{item.image}"]
-    .name
-      = item.name
+main.pages-home
+  .swiper-container.home-swiper
```

Comment:
Let's maybe keep it simple and hold off on adding a swiper for the time being?  

The whole navigation is already a swiper so I don't know if we should nest them.